### ISSUE TITLE
Not Functioning in Plasma 6

### ISSUE THREAD

**Issue by nellePoint** [nclarius/KWin-window-positioning-scripts#27]
had this installed for a while and noticed it wasnt working in plasma 6, so i attempted to reinstall.

Plasma just says it was successfully installed and doesnt list the script.

**Comment by nclarius** [comment]
Yes, this needs to be adapted to still work with Plasma 6. Unfortunately there is currently no complete guide for developers how to do this so I expect it will take me a few weeks.

**Comment by nclarius** [comment]
Which of the scripts is it that you're using? (There are several ones in this repository.)

**Comment by nclarius** [comment]
Also no guarantee that it will still work in Plasma 6 at all because a lot has changed under the hood in the window manager. But I'm planning to at least try.

**Comment by nellePoint** [comment]
Understandable

**Comment by teekoucha** [comment]
There were breaking KWin Script API changes.
change `workspace.clientAdded` to `workspace.windowAdded`
and `window.geometry` to `window.frameGeometry` in main.js
then it should work with Plasma 6 (At least it works on my side)
Just couldn't wait for an update because I can't work without this great plugin 😉

**Comment by technewsguy** [comment]
> There were breaking KWin Script API changes. change `workspace.clientAdded` to `workspace.windowAdded` and `window.geometry` to `window.frameGeometry` in main.js then it should work with Plasma 6 (At least it works on my side) Just couldn't wait for an update because I can't work without this great plugin 😉
I'm not familiar with the items that need to be changed to make this work. Please let us know where you went to make those changes.
